src/main/java/com/tradeshift/flagr/Flagr.java

```diff
@@ -73,7 +73,8 @@ public EvaluationResponse evaluate(EvaluationContext context) {
         EvaluationResponse evalResponse = deserializeResponse(responseBody);
 
         // when the id is null it means Flagr couldn't find it.
-        if (evalResponse.getFlagID() == null) {
+        // when the variant is null, the flag is disabled.
+        if (evalResponse.getFlagID() == null || evalResponse.getVariantKey() == null) {
             String message = evalResponse.getEvalDebugLog().getMsg();
             throw new FlagrException(
```

Comment:
Is this really an exception??

Reply:
So there is two cases in one here. 
- Flag not found
- Flag did not evaluate to a variant key

I maybe the "flag not found" case is an exception. I don't think the other case is an exception.

Reply:
I'll fix that.

Reply:
I would expect an exception to be exceptional, not sure searching for a flag that doesn't exists is that...
Semantics in the end I guess.. and then how you handle errors in Java :|

Reply:
@sorenmat An empty `EvaluationResponse` would also make sense. Should we go for that instead?